release-github/action.yml

```diff
@@ -173,12 +190,56 @@ runs:
       shell: bash
       run: ls -R dist/
 
+    - name: "Install toml"
+      shell: bash
+      run: |
+        python -m pip install --upgrade pip toml==${{ inputs.toml-version }}
+
+    - name: "Get CHANGELOG file location"
+      shell: python
+      run: |
+        import sys
+        sys.path.insert(1, '${{ github.action_path }}/../doc-changelog/')
+
+        from parse_pr_title import get_towncrier_config_value, save_env_variable
+
+        changelog_loc = get_towncrier_config_value("filename")
+        save_env_variable("CHANGELOG_LOC", changelog_loc)
+
+    - name: "Release to GitHub"
+      uses: softprops/action-gh-release@v1
+      env:
+        REPO_NAME: ${{ github.event.repository.name }}
+      permissions:
+        contents: write
+      if: inputs.dry-run == 'false' && env.CHANGELOG_LOC != ""
+      with:
+        fail_on_unmatched_files: false
+        generate_release_notes: true
```

Comment:
Should `generate_release_notes` be false when using the changelog file? This is the description of the input:

Whether to automatically generate the name and body for this release. If name is specified, the specified name will be used; otherwise, a name will be automatically generated. If body is specified, the body will be pre-pended to the automatically generated notes. See the [GitHub docs for this feature](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes) for more information

I think it should be true for the sake of the name being automatically generated?

Reply:
Agreeing